src/components/menus/LocalDocsMenu.ts

```
@@ -0,0 +1,683 @@
+        super(editor, {
+            icon: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16" fill="currentColor">
+                   <path d="M19 5V19H5V5H19ZM21 5C21 3.89543 20.1046 3 19 3H5C3.89543 3 3 3.89543 3 5V19C3 20.1046 3.89543 21 5 21H19C20.1046 21 21 20.1046 21 19V5ZM7 7H17V9H7V7ZM7 11H17V13H7V11ZM7 15H13V17H7V15Z"></path>
+                   </svg>`,
+            tipPosition: 'bottom-right',
+            noBorder: true
+        });
+        
+        this.editor = editor;
+        
+        // Try to get access to the LocalDocsManager, but with safety checks
+        if (editor && editor.innerEditor && editor.innerEditor.extensionManager) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)